assets/constants.js

```diff
@@ -609,5 +609,7 @@ var constants = {
   // eslint-disable-next-line max-len
   "DEFAULT_TWITTER_SHARE_MESSAGE_EDITOR": "Check out this interactive lesson I created on Oppia - a free platform for teaching and learning!",
 
+  "GCS_RESOURCE_BUCKET_NAME": "None-resources",
```

Comment:
Please add a comment to explain this value.

The release process also needs to be updated to include a change to this value. Please work with @ankita240796 to do that. It needs to be done automatically by the deploy.py script and it is important that this PR includes the changes for that script as well.

Reply:
> Please add a comment to explain this value.

Done.

>The release process also needs to be updated to include a change to this value. Please work with @ankita240796 to do that. It needs to be done automatically by the deploy.py script and it is important that this PR includes the changes for that script as well.

The value needs to be set manually or hardcoded in the release scripts.

Reply:
Hi @Jamesjay4199, I think you can use `app_identity_services.get_gcs_resource_bucket_name` in the `deploy.py` script to update the `GCS_RESOURCE_BUCKET_NAME` value in constants during deployment for release. You can add a new function which is called in  `_execute_deployment` function in the deploy script (in the start of the function before other steps) to update the value in constants. Does that sound good?

Reply:
After more discussion, we resolved to concatenate the resources suffix to the app name.

PTAL @ankita240796

Reply:
Looks good now @Jamesjay4199!